app/Party.php

```diff
@@ -627,9 +627,18 @@ public function getEventStats($emissionRatio)
         $fixed_devices = 0;
         $repairable_devices = 0;
         $dead_devices = 0;
+        $devices_powered = 0;
+        $devices_unpowered = 0;
 
         if ( ! empty($this->allDevices)) {
             foreach ($this->allDevices as $device) {
+                error_log("{$device->name} {$device->deviceCategory->powered}");
```

Comment:
Presume this just for debug purposes while still in dev?
(Just in case it's for actual logging, we used the Log::error facade which logs to Laravel's logfile - and allows logs to other channels too)

Reply:
Yes, debug only.  Will remove.